### PR TITLE
Issue #744 Tuya Curtain Motor Updates

### PR DESCRIPTION
**Pull request by rednus**
Clean up of code in init.py and also added device signature for _TZE200_rddyvrci

### CODE REVIEW

**Comment by puddly** [comment]
I'll make a PR for fixing the documentation but all of the formatting issues that you're running into are handled automatically by the Git pre-commit hook:

```console
$ pip install pre-commit
$ pre-commit install  # installs itself as a git pre-commit hook
$ # Make your changes
$ git commit -a  # will automatically format your code
```

**Comment by rednus** [comment]
> I'll make a PR for fixing the documentation but all of the formatting issues that you're running into are handled automatically by the Git pre-commit hook:
> 
> ```
> $ pip install pre-commit
> $ pre-commit install  # installs itself as a git pre-commit hook
> $ pre-commit run -a  # auto-format your existing code, only needs to be run once
> $ # Make your changes
> $ git commit -a  # will automatically format your code
> ```

Hi I will do it now.. thank you

File: zhaquirks/tuya/__init__.py [hunk 1]
```diff
@@ -13,31 +13,94 @@
...
 COVER_EVENT = "cover_event"
+ATTR_COVER_POSITION = 0x0008
+ATTR_COVER_DIRECTION = 0x8001
+ATTR_COVER_INVERTED = 0x8002
+# For most tuya devices 0 = Up/Open, 1 = Stop, 2 = Down/Close
+TUYA_COVER_COMMAND = {
+    "_TZE200_zah67ekd": {0x0000: 0x0000, 0x0001: 0x0002, 0x0002: 0x0001},
+    "_TZE200_fzo2pocs": {0x0000: 0x0000, 0x0001: 0x0002, 0x0002: 0x0001},
+    "_TZE200_xuzcvlku": {0x0000: 0x0000, 0x0001: 0x0002, 0x0002: 0x0001},
+    "_TZE200_rddyvrci": {0x0000: 0x0000, 0x0001: 0x0002, 0x0002: 0x0001},
```
Comment (jasmin-mistry) [inline zhaquirks/tuya/__init__.py:81]: Please can we change this to the below for `_TZE200_rddyvrci`

`"_TZE200_rddyvrci": {0x0000: 0x0002, 0x0001: 0x0001, 0x0002: 0x0000},`

I have tested and the above parameter mapping works.

**Comment by blauret** [comment]
This looks good to me. Can be merged in my opinion. this is a welcome improvement on the original code.

**Pull request merged**